lib/modem_info/modem_info.c

```diff
@@ -888,6 +915,22 @@ int modem_info_get_rsrp(int *val)
 	return 0;
 }
 
+int modem_info_get_connectivity_stats(int *tx_kbytes, int *rx_kbytes)
+{
+	if (tx_kbytes == NULL || rx_kbytes == NULL) {
+		return -EINVAL;
+	}
+
+	int ret = nrf_modem_at_scanf(AT_CMD_XCONNSTAT, "%%XCONNSTAT: %*d,%*d,%d,%d,%*d,%*d",
+				     tx_kbytes, rx_kbytes);
```

Comment:
For my info, does the format specifier `%*d` allow us to skip providing that a destination for that field?

Reply:
Yes! I also learned this while working on this. A `*` in a format specification for `scanf` indicates that the data is to be read from the stream but ignored, i.e. it is not stored a corresponding argument

Reply:
Very nice, thanks!